loony_dev/web/services.py

```
@@ -305,6 +310,251 @@ def list_sessions(base_dir: Path) -> list[SessionView]:
+            for pipeline in Pipeline.discover(repo):
+                pipelines.append(_pipeline_view(pipeline, repo_name))
+            # ``discover`` already populated the tick-cached open-PR list, so this
+            # is a cache hit — no extra ``gh`` call.
+            open_prs = len(PullRequest.list_open(repo=repo))
+            open_issues = _count_open_issues(repo)
+            repos.append(
+                RepoGitHubView(
+                    repo=repo_name, open_issues=open_issues, open_prs=open_prs, ok=True,
+                )
+            )
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)